### PR TITLE
Wire mapping fix

### PR DESCRIPTION
**Pull request by lillian542**
Remove custom wire logic to avoid ambiguity; wire `5` is always qubit 5, wire `3` is always qubit 3, and there is no wire `'a'`

### CODE REVIEW

File: src/braket/pennylane_plugin/translation.py [hunk 1]
```diff
@@ -445,10 +445,10 @@ def _(op: ParametrizedEvolution, _parameters, device=None):
     # The driven wires aren't the same as `op.wires` as `op.wires` contains
     # all device wires due to interaction term.
     pulse_wires = qml.wires.Wires.all_wires([pulse.wires for pulse in pulses])
-    mapped_wires = pulse_wires.map(device.wire_map)
+    frames = {w: device._device.frames[f"q{w}_drive"] for w in pulse_wires}
```
Comment (mudit2812) [inline src/braket/pennylane_plugin/translation.py]: With this change, we don't need the `BraketDevice` instance for the translation, just the `AwsDevice` instance, so we could change that as well.

Comment (lillian542) [reply src/braket/pennylane_plugin/translation.py]: Good point, thanks! I'll update that. Nicer not to have two different devices floating around :)

File: src/braket/pennylane_plugin/translation.py [hunk 2]
```diff
@@ -182,6 +182,7 @@ def translate_operation(
             p.numpy() if isinstance(p, qml.numpy.tensor) else p for p in operation.parameters
         ]
     device = kwargs.get("device", None)
+    print(type(device))
```
Comment (lillian542) [inline src/braket/pennylane_plugin/translation.py]:
```suggestion
```

**Review by Qottmann — APPROVED** [review]
Amazingly quick turnaround, thanks @lillian542 !

**Pull request merged**